src/Symfony/Component/Mailer/Bridge/Sendgrid/Tests/Transport/SendgridApiTransportTest.php

```diff
@@ -192,4 +192,31 @@ public function testReplyTo()
         $this->assertArrayHasKey('email', $payload['reply_to']);
         $this->assertSame($replyTo, $payload['reply_to']['email']);
     }
+
+    public function testEnvelopeSenderAndRecipients()
+    {
+        $from = 'from@example.com';
+        $to = 'to@example.com';
+        $envelopeFrom = 'envelopefrom@example.com';
+        $envelopeTo = 'envelopeto@example.com';
+        $email = new Email();
+        $email->from($from)
+            ->to($to)
```

Comment:
Can you also add some cc and bcc to be sure they don't end up in to?

Reply:
Added cc + bcc, + a count on the to items